<ui version="4.0" >
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="findLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>91</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_findInput" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>91</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Replace With:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_replaceInput" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="ui_matchCaseCheckBox" >
       <property name="text" >
        <string>Match Case</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="ui_buttonClose" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="ui_buttonReplace_All" >
         <property name="text" >
          <string>Replace All</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_buttonReplace" >
         <property name="text" >
          <string>Replace</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_buttonFind" >
         <property name="text" >
          <string>Find</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ui_buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>49</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel" >
     <x>173</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_findInput</sender>
   <signal>returnPressed()</signal>
   <receiver>ui_buttonFind</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>322</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_replaceInput</sender>
   <signal>returnPressed()</signal>
   <receiver>ui_buttonReplace</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
